README.md

```diff
@@ -21,11 +21,11 @@ and [J. R. Johansson](https://github.com/jrjohansson)
 [![PyPi Downloads](https://img.shields.io/pypi/dm/qutip?label=downloads%20%7C%20pip&logo=PyPI)](https://pypi.org/project/qutip)
 [![Conda-Forge Downloads](https://img.shields.io/conda/dn/conda-forge/qutip?label=downloads%20%7C%20conda&logo=Conda-Forge)](https://anaconda.org/conda-forge/qutip)
 
-QuTiP is open-source software for simulating the dynamics of closed and open quantum systems.
-The QuTiP library uses the excellent Numpy, Scipy, and Cython packages as numerical backend, and graphical output is provided by Matplotlib.
+QuTiP is an open-source software for simulating the dynamics of closed and open quantum systems.
+The QuTiP library uses Numpy, Scipy, and Cython packages as numerical backends, and Matplotlib for graphical output.
```

Comment:
```suggestion
QuTiP is open-source software for simulating the dynamics of closed and open quantum systems.
It uses the excellent Numpy, Scipy, and Cython packages as numerical backends, and graphical output is provided by Matplotlib.
```
"software" is an uncountable noun and so does not take an article. Praising numpy, SciPy and Cython shows we're grateful to those projects and sets a good tone, so I'd like to keep it. Using a slightly different phrasing when mentioning Matplotlib makes the sentence less repetitive, so I'd like to keep it that way.